pyproject.toml

```diff
@@ -14,14 +14,22 @@ repository = "https://github.com/autotraderuk/dbt-dry-run"
 dbt-dry-run = "dbt_dry_run.__main__:main"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = ">=3.8,<3.13"
 agate = ">=1.7.0,<1.10"
 google-cloud-bigquery = "^3"
 pydantic = "^1.9.0"
 tenacity = "^8.2"
 networkx = ">=2.3,<4.0"
 pyyaml = "~6"
 typer = "~0"
+numpy = [
```

Comment:
I don't think this package has a dependency on numpy or pandas directly. Should this be set in the dev dependencies instead?

Reply:
I was wrong, I'll set it up again in dev dependencies.

numpy 1.24 that has distutils doesn't support python 3.12 because we can't use distutils with python 3.12.

python 3.8 is supported by pandas up to 2.0.3.
We need to build pandas locally, because pandas for python3.12 does not exist in pypi with versions up to 2.1.0.
If we can accept local builds taking longer, no need to specify this. (In my environment it took 9 minutes.)

Reply:
fixed https://github.com/autotraderuk/dbt-dry-run/pull/72/commits/9024a237cb522a21812a89dd48126a2ec09c3471